packages/plasma-b2c/src/components/Badge/Badge.update-test.component-test.tsx

```diff
@@ -115,10 +124,10 @@ describe('plasma-b2c: Badge', () => {
         cy.matchImageSnapshot();
     });
 
-    it('[PLASMA-] Badge: customBackroundColor, customColor', () => {
+    it('[PLASMA-1652] Badge: customBackroundColor, customColor', () => {
```

Comment:
_minor_

опечатка в background
```suggestion
    it('[PLASMA-1652] Badge: customBackgroundColor, customColor', () => {
```

Reply:
исправил